java/code/src/com/redhat/rhn/frontend/xmlrpc/system/monitoring/SystemMonitoringHandler.java

```diff
@@ -56,7 +56,7 @@ public SystemMonitoringHandler(FormulaManager formulaManagerIn) {
      *     $EndpointInfoSerializer
      *   #array_end()
      */
-    public List<EndpointInfo> listEndpoints(User loggedInUser, List<Long> systemIDs) {
+    public List<EndpointInfo> listEndpoints(User loggedInUser, List<Number> systemIDs) {
```

Comment:
The other XML-RPC API endpoints use `Integer` rather than `Number`, it would be nice to align with them

Reply:
Thanks for the review. Done.